kafka-scheduler/src/main/java/com/mesosphere/dcos/kafka/config/KafkaSchedulerConfiguration.java

```diff
@@ -181,15 +188,26 @@ public String toString() {
         @Override
         public KafkaSchedulerConfiguration parse(byte[] bytes) throws ConfigStoreException {
             try {
-                return JsonUtils.MAPPER.readValue(bytes, KafkaSchedulerConfiguration.class);
+
+                final ObjectMapper mapper = Jackson.newObjectMapper()
+                        .registerModule(new GuavaModule())
+                        .registerModule(new JavaTimeModule())
+                        .registerModule(new Jdk8Module());
+                return  mapper.readValue(bytes, KafkaSchedulerConfiguration.class);
+                //return JsonUtils.MAPPER.readValue(bytes, KafkaSchedulerConfiguration.class);
             } catch (Exception e) {
                 throw new ConfigStoreException(e);
             }
         }
     }
 
     @Override
-    public String toJsonString() throws Exception {
-        return new ObjectMapper().writerWithDefaultPrettyPrinter().writeValueAsString(this);
+    public String toJsonString() throws ConfigStoreException {
+        //return new ObjectMapper().writerWithDefaultPrettyPrinter().writeValueAsString(this);
+        try {
+            return SerializationUtils.toJsonString(this);
+        } catch (IOException e) {
+            return "";
```

Comment:
Is failing quietly the right behavior here? Getting an empty string could be mystifying for someone down the line.